lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2323,26 +2323,26 @@ ParseResult FIRStmtParser::parsePrintf() {
   StringRef formatStringRef(formatStrUnescaped);
   if (formatStringRef.startswith("cover:")) {
     APInt constOne(1, 1, false);
-    auto constTrue =
+    Value constTrue =
         builder.create<ConstantOp>(UIntType::get(getContext(), 1), constOne);
-    builder.create<CoverOp>(clock, condition, constTrue, formatStrUnescaped,
-                            "");
+    builder.create<CoverOp>(clock, condition, constTrue, formatStrUnescaped, "",
+                            true);
```

Comment:
Plz put a /*Foo=*/ comment before the true so we know what is being set. (also below)